lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -114,25 +114,25 @@ define((require, exports, module) ->
     @$rules =
       'start': [
         beginRule(LIST_REGEX, LIST_STATE)
-        beginRule(EQUATIOТ_REGEX, EQUATION_STATE)
+        beginRule(EQUATION_REGEX, EQUATION_STATE)
 
-        endRule(EQUATIOТ_REGEX)
+        endRule(EQUATION_REGEX)
         endRule(LIST_REGEX)
 
       ]
-      EQUATION_STATE: [
-        beginRule(EQUATIOТ_REGEX, EQUATION_STATE)
+      'equation': [
```

Comment:
because our backend doesn't support constant in this place.
You can try undo about two commits and see the console